src/components/v5/common/ObjectiveBox/ObjectiveBox.tsx

```diff
@@ -27,7 +29,12 @@ const ObjectiveBox: FC<ObjectiveBoxProps> = ({ objective }) => {
         {objective?.title || formatMessage(MSG.noObjectiveBoxTitle)}
       </h6>
       <p className="text-sm text-gray-600 mb-[1.125rem]">
-        {objective?.description || formatMessage(MSG.noObjectiveBoxDescription)}
+        {(objective &&
```

Comment:
maybe we can rather use a ternary here

Reply:
Ye you are right, I've used ternary here